.github/markdownlint.yml

```diff
@@ -1,5 +1,12 @@
 # Markdownlint configuration file
-default: true,
+default: true
 line-length: false
 no-duplicate-header:
     siblings_only: true
+no-inline-html:
+    allowed_elements:
+        - img
+        - p
+        - kbd
```

Comment:
what is this element?

Reply:
Not sure either. If I start a new template, they are not included, which is strange right.
We haven't added them, so I'm assuming they have been added automatically.
They also appear here: https://github.com/nf-core/rnaseq/blob/master/.github/markdownlint.yml